conf.py

```diff
@@ -201,16 +201,16 @@ def all_keybinds_documented():
 # short alias names to a base URL and a prefix.
 # See http://sphinx-doc.org/ext/extlinks.html
 extlinks = {
-    'wiki': ('https://dwarffortresswiki.org/%s', ''),
+    'wiki': ('https://dwarffortresswiki.org/%s', '%s'),
```

Comment:
This appears to be a breaking change. Sphinx 1.8-built docs (like the ones [built for this PR](https://dfhack--2193.org.readthedocs.build/en/2193/docs/_auto/fix.html)) don't interpolate the `%s` and look like this:

![image](https://user-images.githubusercontent.com/3719547/172774248-471cb3bd-79a0-43b7-b61a-a685518f1dee.png)

I'm confused exactly when this was introduced. Based on https://github.com/sphinx-doc/sphinx/pull/8898, it sounds like there should have been a warning before the breaking change, but I don't recall seeing one. That issue is also marked with the `4.1.0` milestone, but is linked as a breaking change in 4.0.0 at https://www.sphinx-doc.org/en/master/changes.html (and I've definitely built successfully with 4.x, most recently 4.5.0).

Anyway, we can probably add the `%s` programmatically ourselves, if really needed. Until we decide that we need to take advantage of it not always being at the end of the string...

Reply:
Yes, it seems like I only checked extlinks that came with explicit captions (e.g.
```
:dffd:`mycaption <12345>`
```
The extlinks that use the default caption are indeed broken with <sphinx-5. It looks like they added the deprecation warning of the old syntax in the same version that they enabled parsing of the new syntax.

I'll close this PR and just pin my local version of sphinx to 4.X

Reply:
I forgot to follow up. I tested this PR locally and I actually did see correct rendering with Sphinx 4.5, so somehow it supports formats with and without %s. Or it's also possible that my test was flawed.

Pinning to <5 is fine by me. The reason it's pinned to <4.4 in GitHub actions is because of an overly-agressive extlinks warning in 4.4, but 4.5 fixed that.